### DISCUSSION TITLE
Using fuzzy search {%} Question

### DISCUSSION THREAD

**Discussion by S3BzA** [sherlock-project/sherlock#1823] [Q&A] [answered]
Hi Sherlock community,

I'm uncertain how to use the similar username feature.

For example, the --help says that you can use {%} to find alike usernames, but i don't actually know how to implement that. I assume you treated it like the LIKE comparison in MYSQL where you go `$ python3 sherlock %username%` but sherlock takes that argument as a literal and finds usernames that are an exact match to the string "%username%".

How is it used? and additionally what does it mean by using _ - . 

The snippet to which I refer:

```
positional arguments:
  USERNAMES             One or more usernames to check with social networks.
                        Check similar usernames using {%} (replace to '_', '-', '.').
```

**Comment by p20f** [comment, marked as answer]
You should use {?} instead of {%}

**Reply by jmacman123** [reply]
Could you write an example still a bit confused

**Reply by p20f** [reply]
```
python3 sherlock user{?}name
``` 
It checks for three different users: 
- user_name
- user-name
- user.name

It is the same as writing:
```
python3 sherlock user_name user-name user.name
```
